### DISCUSSION TITLE
Is there a way to forward logs from syslog-ng to journald?

### DISCUSSION THREAD

**Discussion by avi9526** [syslog-ng/syslog-ng#4757] [Q&A] [answered]
Is there a way to forward logs from syslog-ng to journald? I want to receive logs from remote hosts with syslog but keep it in journald.

**Comment by bazsi** [comment, marked as answer]
It would be possible to write a native journal destination, but we don't have that at the moment.

One potential solution is to send messages to /dev/log, just make sure it is not forwarded back to syslog-ng.

To do that you need the unix-dgram() destination driver.

E.g. 

destination systemd {
  unix-dgram("/dev/log");
};
